rotkehlchen/chain/ethereum/modules/uniswap/uniswap.py

```diff
@@ -248,12 +252,16 @@ def _get_balances_graph(
 
                 for token in token0, token1:
                     # Get the token <EthereumToken> or <UnknownEthereumToken>
-                    asset = get_ethereum_token(
-                        symbol=token['symbol'],
-                        ethereum_address=to_checksum_address(token['id']),
-                        name=token['name'],
-                        decimals=int(token['decimals']),
-                    )
+                    try:
+                        asset = get_ethereum_token(
+                            symbol=token['symbol'],
+                            ethereum_address=deserialize_ethereum_address(token['id']),
```

Comment:
try/except only around deserialize_ethereum_address. Also as with above, I think this is a fatal error. Raise a `RemoteError`. Graph giving us invalid address data is probably pretty bad.

Reply:
But looking at the functions that call this one, none of them handle the RemoteError I believe

Reply:
At line 230 this function already raises a `RemoteError`.

And it is handled. These end up in uniswap.get_balances which is called from the api here: https://github.com/rotki/rotki/blob/8833d0f327a3039cf338d20af03951738f4a0d49/rotkehlchen/api/rest.py#L2215-L2223

and handles remote errors here:

https://github.com/rotki/rotki/blob/8833d0f327a3039cf338d20af03951738f4a0d49/rotkehlchen/api/rest.py#L2027-L2031